core/module.h

```diff
@@ -137,7 +165,43 @@ class ModuleBuilder {
 
   CommandResponse RunInit(Module *m, const google::protobuf::Any &arg) const;
 
+  /*!
+   * Connects two modules together in `module_graph_`.
+   */
+  static int AddEdge(const std::string &from, const std::string &to);
```

Comment:
If the return is just "success" or "failure" then prefer to use a bool and return true upon success.